jest.config.js

```diff
@@ -21,6 +21,7 @@ module.exports = {
     "/test/helpers/",
     "<rootDir>/test/warning\\.js",
     "<rootDir>/build/",
+    "<rootDir>/.history/",
```

Comment:
What is `.history` from?

Reply:
VS Code Extension - https://marketplace.visualstudio.com/items?itemName=xyz.local-history

Reply:
Since the `.history` folder is not generated by Babel build scripts, can you add a comment about why `.history` is excluded (e.g. it is used by a VSCode plugin). Ditto for the `.gitignore`.

Reply:
done